spring-kafka/src/main/java/org/springframework/kafka/retrytopic/DestinationTopic.java

```diff
@@ -219,6 +219,15 @@ public long delay() {
 			return this.delayMs;
 		}
 
+		/**
+		 * Returns the number of partitions the
```

Comment:
The method javadoc has to be imperative: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting
And `@since` ?